app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:user_name, :first_name, :last_name, :picture, :pass, :bio, :sport_type, :role])
+  end
```

Comment:
I'm sure I don't know this just because I've never used devise before, but is the parameter_sanitizer like doing strong params?